Sourcery/Models/Enum.swift

```diff
@@ -51,6 +51,7 @@ class Enum: Type {
     internal(set) var rawType: String? {
         didSet {
             if let rawType = rawType {
+                hasRawType = true
```

Comment:
I've changed a bit the logic around `hasRawType`, now we set it right in the `init`, as we already should have directly inherited types list there.